hazelcast/src/main/java/com/hazelcast/cache/impl/AbstractCacheRecordStore.java

```diff
@@ -341,7 +341,7 @@ public void onEvict(Data key, R record) {
 
     protected void invalidateEntry(Data key, String source) {
         if (isInvalidationEnabled()) {
-            cacheService.sendInvalidationEvent(name, toHeapData(key), source);
+            cacheService.sendInvalidationEvent(getName(), toHeapData(key), source);
```

Comment:
As far as I see no, this is from Marko's commit: https://github.com/mtopolnik/hazelcast/commit/ad6a0572d3ee9b95bb20868ec22113f61b07a6ab

So, I don't know the reasoning :)